tests/test_data/json_schema/iSamplesSchemaCore1.0.json

```diff
@@ -235,7 +235,15 @@
                 },
                 "result_time": {
                     "description": "Date on which the sample was collected.",
-                    "format": "date",
+                    "anyOf": [
+                        {
+                            "format": "date"
+                        },
+                        {
+                            "format": "date-time"
```

Comment:
I'm not certain this is correct.  It passes, though.

Reply:
@datadavev if you have thoughts here this is what I'm unsure about

Reply:
Per discussion with @datadavev in this morning's standup, this is correct: https://json-schema.org/understanding-json-schema/reference/string#built-in-formats